libraries/standard/json/src/json.c

```diff
@@ -423,7 +445,11 @@ static bool_ skipString( const char * buf,
                          size_t max )
 {
     bool_ ret = false;
-    size_t i = *start;
+    size_t i;
+
+    assert( ( buf != NULL ) && ( start != NULL ) && ( max > 0U ) );
```

Comment:
do we really need the `U` on `0`?

Reply:
MISRA 
Rule 10.4 | Both operands of an operator in which the usual arithmetic conversions are performed shall have the same essential type category

Some places I can use a 0, but some places I need 0U, based on coverity scans.  I did change assignments from 0U to 0, since that generated no MISRA violations.